lib/matplotlib/gridspec.py

```diff
@@ -448,10 +519,10 @@ def get_topmost_subplotspec(self):
 
     def __eq__(self, other):
         # other may not even have the attributes we are checking.
-        return ((self._gridspec, self.num1, self.num2)
-                == (getattr(other, "_gridspec", object()),
-                    getattr(other, "num1", object()),
-                    getattr(other, "num2", object())))
+        return ((self._gridspec, self.num1, self.num2) ==
+                (getattr(other, "_gridspec", object()),
```

Comment:
something when wrong with the alignment of the lines below (and I prefer the operator at the beginning of the line but that's fine)

Reply:
I did 
```python
        return ((self._gridspec, self.num1, self.num2)
                == (getattr(other, "_gridspec", object()),
                    getattr(other, "num1", object()),
                    getattr(other, "num2", object())))
```